mails/_partials/order_conf_product_list.txt

```diff
@@ -0,0 +1,20 @@
+{foreach $list as $product}
+						{$product['reference']}
```

Comment:
really weird indent

Reply:
I agree, but it's a txt template from the legacy so I prefer to keep it as is

Reply:
:(